HeliPort/AppDelegate.swift

```diff
@@ -31,8 +28,17 @@ class AppDelegate: NSObject, NSApplicationDelegate, NSMenuDelegate {
         StatusBarIcon.statusBar = statusBar
     }
 
-    func applicationWillTerminate(_ aNotification: Notification) {
-        // Insert code here to tear down your application
-    }
+    private func checkDriver() {
+        var drv_info = ioctl_driver_info()
+        _ = ioctl_get(Int32(IOCTL_80211_DRIVER_INFO.rawValue), &drv_info, MemoryLayout<ioctl_driver_info>.size)
 
+        let version = cTupleToString(tuple: drv_info.driver_version)
+        let interface = cTupleToString(tuple: drv_info.bsd_name)
```

Comment:
why not use
```swift
let version = String(cString: &drv_info.driver_version.0)
```

Reply:
It is my first time doing C interop, did not know about this. I will try it.